### ISSUE TITLE
Integration with Gravity Forms or other Form Builder Extension

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#221]
```
What functionality do you want to see?


Do you have examples of similar functionality on other sites or in other
plugins? (Please share URLs below)


Please provide any additional information below.

It would be great to have the ability for Gravity forms or another form builder 
extension to be able to interact with the fields/tables built by the custom 
content type manager so that a front end form is able to populated the custom 
fields.
```

Original issue reported on code.google.com by `silberma...@gmail.com` on 17 Oct 2011 at 12:34

**Comment by GoogleCodeExporter** [comment]
```
Interesting idea.  I'm working on something like this for 0.9.5 with the 
integration of my Summarize Posts plugin.  Note that the CCTM plugin uses the 
standard WP tables -- no new database tables are created.  This relies on the 
existing WP architecture, so if there is some solution that works already with 
existing WP posts or pages, then it could be easily adapted to work with the 
CCTM since in WP, the only difference between a "post", "page", or an 
"attachment" is the value in a single column: post_type.
```

Original comment by `fireproofsocks` on 17 Oct 2011 at 12:54
- Added labels: **Priority-Medium**, **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
This is available in 0.9.5: use the SP_Post class to create posts and their 
custom fields through a simple all-in-one API method.  I'll update the Wiki 
here for this use-case.

0.9.5 also integrates with my Summarize Posts plugin, so there's a GetPostForm 
class that can be used to create dynamic, tokenized search forms.  Again, wiki 
docs are forthcoming.
```

Original comment by `fireproofsocks` on 27 Nov 2011 at 7:36
- Changed state: **Fixed**
